eureka_ml_insights/data_utils/data.py

```diff
@@ -620,7 +620,7 @@ def _load_dataset(self) -> pd.DataFrame:
                 hf_dataset = load_dataset(self.path, task, cache_dir=self.cache_dir, split=self.split)
```

Comment:
theres another "split" on line 616

Reply:
Thank you for catching this. I fixed it and added a pipeline test for a pipeline that would have failed because of this but didn't have test.